portal-web/test/functional/com/liferay/portalweb/macros/PublicationsChanges.macro

```diff
@@ -148,39 +148,22 @@ definition {
 	}
 
 	macro moveChangeToAnotherPublication {
-		Click.clickAtNotVisible(
-			changeOrder = ${changeOrder},
+		Click(
 			key_changeCategory = ${changeCategory},
+			key_changeOrder = ${changeOrder},
 			key_changeTitle = ${changeTitle},
-			locator1 = "PublicationsChanges#CHANGES_CHECK_BOX");
+			locator1 = "PublicationsChanges#REVIEW_CHANGE_LISTING");
+
+		Click(locator1 = "PublicationsChanges#REVIEW_CHANGE_DROPDOWN");
 
 		Click(
-			ariaLabel = "Show Move Changes Modal",
-			locator1 = "Button#ANY_WITH_ARIA_LABEL");
+			key_menuItem = "Move Changes",
+			locator1 = "PublicationsChanges#REVIEW_CHANGE_DROPDOWN_ITEM");
 
 		Select(
-			ariaLabel = "Select the publication you would like to move your changes to.",
+			ariaLabel = "Publication",
 			locator1 = "Select#ANY_ARIA_LABEL",
 			value1 = ${targetPublicationName});
```

Comment:
I left the button click outside of the macro because for `CanMoveChangeBetweenPublications`, we wanted the Assert statements before moving the changes, so I needed to leave a gap.
Instead, I added a conditional so you can choose whether or not the button will be pressed via the macro (by default it will press). Hopefully that helps!